<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="ezs013" name="com.eDocs.SanityTesting.EquipmentGroup" tests="4" failures="1" timestamp="28 May 2018 05:45:11 GMT" time="68.966" errors="0">
  <testcase name="CreateEquipmentGroup" time="19.818" classname="com.eDocs.SanityTesting.EquipmentGroup"/>
  <testcase name="CreateEquipmentGroup" time="22.643" classname="com.eDocs.SanityTesting.EquipmentGroup"/>
  <testcase name="EditEquipmentGroup" time="18.190" classname="com.eDocs.SanityTesting.EquipmentGroup"/>
  <testcase name="SingleDeleteEquipmentGroup" time="8.315" classname="com.eDocs.SanityTesting.EquipmentGroup">
    <failure type="java.lang.AssertionError" message="expected [Equipment Group was deleted successfully] but found [Equipment Group deleted successfully]">
      <![CDATA[java.lang.AssertionError: expected [Equipment Group was deleted successfully] but found [Equipment Group deleted successfully]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.eDocs.SanityTesting.EquipmentGroup.SingleDeleteEquipmentGroup(EquipmentGroup.java:408)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SingleDeleteEquipmentGroup -->
</testsuite> <!-- com.eDocs.SanityTesting.EquipmentGroup -->
